app/views/collections/julia-child.md

```diff
@@ -29,16 +29,14 @@ from the set, and/or screen grabs from the programs.
 
 ## Links
 
-[Julia Child programs (downloadable PDF)](/TODO)
-
 [Julia Child Foundation](http://www.juliachildfoundation.org)
 
-[A la Carte Productions (Produced later Julia Child Productions post 1984)](/TODO)
+[A la Carte Productions (Produced later Julia Child Productions post 1984)](http://www.alacartetv.com/html/jnj/jnj.htm)
```

Comment:
@afred: no, I think that means something different. Maybe just
>
A la Carte Productions (Julia Child programs after 1984)